test/chplcheck/Unused.chpl

```diff
@@ -0,0 +1,5 @@
+module Unused {
+  proc myProc(A, B) {
+    for i in 1..10 do writeln(A);
```

Comment:
This should check for nested loops, too:

```Chapel
for (i, j) in 1..10 do writeln(i); // should warn for j
```

Reply:
This case also triggers the ConsecutiveDecls rule, so I exempted TupleDecls from the ConsecutiveDecls traversal.